config/default-disco.js

```diff
@@ -55,7 +55,6 @@ module.exports = {
         'data:',
         amoProdCDN,
         staticHost,
-        'https://www.google-analytics.com',
```

Comment:
On Line 50, there is the same URL suffixed with the js file. I guess we could reuse `analyticsHost` for this value too.